.golangci.toml

```diff
@@ -87,7 +83,7 @@ enable = [
   "dogsled",
   "errcheck",
   # "gochecknoinits",
-  "goconst",
+  # "goconst",
```

Comment:
`mimir` does not event use it. It creates too many false positives. So I disable it.
Mimir config https://github.com/grafana/mimir/blob/main/.golangci.yml

Reply:
Maybe just remove the line instead of commenting it out?